octofree/main.py

```
@@ -32,8 +55,14 @@
 )
 
 def _log_loaded_settings():
+    """Log current configuration settings for debugging."""
     logging.info("Loaded settings:")
-    logging.info(f"  DISCORD_WEBHOOK_URL={os.getenv('DISCORD_WEBHOOK_URL')}")
+    # Mask webhook URL for security - never log the full URL
+    webhook_url = os.getenv('DISCORD_WEBHOOK_URL')
+    if webhook_url:
+        logging.info(f"  DISCORD_WEBHOOK_URL=SET (length: {len(webhook_url)} chars)")
+    else:
+        logging.info(f"  DISCORD_WEBHOOK_URL=NOT SET")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)